packages/mobile/src/screens/update-required-screen/NewVersionPrompt.tsx

```diff
@@ -0,0 +1,65 @@
+import { View } from 'react-native'
+
+import IconDownload from 'app/assets/images/iconDownload.svg'
+import { Button, GradientText, Text } from 'app/components/core'
+import { makeStyles } from 'app/styles'
+
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
+  contentContainer: {
+    paddingTop: spacing(32),
+    paddingBottom: spacing(8),
+    paddingHorizontal: spacing(6)
+  },
+  header: {
+    fontSize: 24,
+    lineHeight: 52,
+    textAlign: 'center',
+    textShadowOffset: { height: 2, width: 0 },
```

Comment:
Fun that we have text shadow, are any of these values we want to standardize? Realizing this was just moved from existing screen though

Reply:
if it's in new design system then yes, otherwise no IMO!

Reply:
word, prob not worth having a whole discussion about it for this pr, but curious what ux thinks long term about gradient text